examples/evtest.rs

```diff
@@ -126,10 +129,20 @@ fn main() {
     }
 
     let path = &args.nth(1).unwrap();
-    let f = File::open(path).unwrap();
+    let mut file = OpenOptions::new()
+        .read(true)
+        .write(true)
+        .custom_flags(libc::O_NONBLOCK)
+        .open(path)
+        .unwrap();
+    let mut buffer = Vec::new();
+    let result = file.read_to_end(&mut buffer);
```

Comment:
This potentially allocates (maybe even allocates a lot) for devices with many unread events. Consider using a fixed size buffer and a loop 
``` rust
let mut buf = [0u8; std::mem::size_of::<input_event>()*20];
while file.read(&mut buf).is_ok() {}
```
or something with file.bytes()

Reply:
Reading in fixed-size sounds better. I'll update the PR.